### DISCUSSION TITLE
Not able to build MuleSoft AI Chain Connector in Windows 11 OS

### DISCUSSION THREAD

**Discussion by tiru** [MuleSoft-AI-Chain-Project/mule-ai-chain-connector#64] [General]
Not able to build MuleSoft AI Chain Connector in Windows 11 OS and getting below Error.

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-jdeps-plugin:3.1.2:jdkinternals (default) on project mulechain-ai-connector: Unable to execute jdeps command: Error while executing process.: Cannot run program "cmd.exe": CreateProcess error=206, The filename or extension is too long -> [Help 1]

[ERROR] Exit code: 1 - The command line is too long.

As workaround, Just remove "Maven-jdeps-plugin" and rebuild MuleSoft AI Chain Connector. Than above two issues got resolved. 

I belive this is not correct way to address above issue. Can you please help why do we need jdeps plugin and how to resolve?

**Comment by amirkhan-ak-sf** [comment]
Please check this: https://github.com/MuleSoft-AI-Chain-Project/mulesoft-ai-chain-connector/issues/58

**Comment by tiru** [comment]
Thank you 👍
